tests/yast2_cmd/yast_ftp_server.pm

```diff
@@ -110,7 +110,7 @@ sub run {
     if (is_sle('15+')) {
         assert_script_run 'yast ftp-server startup socket';
     }
-    if (is_sle('<=12-sp4')) {
+    if (is_sle('<15')) {
```

Comment:
Just to be explicit, I'd personally use:
```suggestion
    if (is_sle('<=12-sp5')) {
```

Reply:
fixed as suggested.